### DISCUSSION TITLE
XCH rewards

### DISCUSSION THREAD

**Discussion by chenus1** [Chia-Network/chia-blockchain#8686] [Support]
I have 675 TB in farming and did not receive any reward for almost one month (since 9/4/2021). It is normal? It is kind of discouraging! I am running win pro 8.1 and 1.2.5 version chia. I checked the debug file and search "time" and attaching the snapshot below. If I plan to join the pool, can someone recommend one? I have multiple harvesters and one farmer, can I join the pool on the farmer machine and it will apply all the harvesters or I need to join the pool in each harvester? Thanks! 
![image](https://user-images.githubusercontent.com/45975235/135726489-0d58331b-a3c4-4270-8bfe-ea2061fd2a6c.png)

**Comment by loppefaaret** [comment]
If the farmer points its plotNFT's at a pool, all subsequent remote harvesters connected to it, will be used to farm for the pool

**Comment by tangdasan** [comment]
welcome join ....

**Reply by loppefaaret** [reply]
> welcome join ....

Do not start running advertisement on our github for your pool

**Comment by BrandtH22** [comment]
@chenus1 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
